frontend/src/components/FileUploader.js

```diff
@@ -47,12 +47,59 @@ export const handleDrop = async (e, reportId, id, onChange, setErrorMessage) =>
   });
 };
 
+export const FileRejections = ({
+  fileRejections,
+}) => {
+  const fileRejectionItems = fileRejections.map(({ file, errors }) => (
+    <li key={file.path}>
+      The file
+      {' '}
+      <strong>{file.path}</strong>
+      {' '}
+      has the following errors:
+      <ul>
+        {errors.map((e) => (
+          <li key={e.code}>{e.code === 'file-too-large' ? 'File is larger than 5 MB' : e.message}</li>
+        ))}
+      </ul>
+    </li>
+  ));
+
+  return (
+    <ul>
+      {
+        fileRejectionItems
+      }
+    </ul>
+  );
+};
+
+FileRejections.propTypes = {
+  fileRejections: PropTypes.oneOfType([
+    PropTypes.arrayOf(
+      PropTypes.shape({
+        errors: PropTypes.arrayOf(
+          PropTypes.shape({
+            code: PropTypes.string.isRequired,
+            message: PropTypes.string.isRequired,
+          }),
+        ),
+      }),
+    ),
+  ]).isRequired,
+};
+
 function Dropzone(props) {
   const { onChange, id, reportId } = props;
   const [errorMessage, setErrorMessage] = useState();
   const onDrop = (e) => handleDrop(e, reportId, id, onChange, setErrorMessage);
+  const maxSize = 5242880;
```

Comment:
Why was 5 mb picked as the limit? Also can we add a comment mentioning this equals 5 mb?